pyaedt/filtersolutions_core/attributes.py

```diff
@@ -0,0 +1,1530 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from ctypes import POINTER
+from ctypes import byref
+from ctypes import c_bool
+from ctypes import c_char_p
+from ctypes import c_int
+from enum import Enum
+
+import pyaedt
+
+
+class FilterType(Enum):
+    """Provides an enum of filter types with associated mathematical formulations.
+
+    Attributes:
+    GAUSSIAN: Represents a Gaussian filter.
+    BESSEL: Represents a Bessel filter.
+    BUTTERWORTH: Represents a Butterworth filter.
+    LEGENDRE: Represents a Legendre filter.
+    CHEBYSHEV_I: Represents a Chevyshev type I filter.
+    CHEBYSHEV_II: Represents a Chevyshev type II filter.
+    HOURGLASS: Represents an hourglass filter.
+    ELLIPTIC: Represents an elliptic filter.
+    Custom, raised-cos, and matched filter types are not available in this release.
```

Comment:
```suggestion
    **Attributes:**
    
    - GAUSSIAN: Represents a Gaussian filter.
     - BESSEL: Represents a Bessel filter.
    - BUTTERWORTH: Represents a Butterworth filter.
    - LEGENDRE: Represents a Legendre filter.
    - CHEBYSHEV_I: Represents a Chevyshev type I filter.
    - CHEBYSHEV_II: Represents a Chevyshev type II filter.
    - HOURGLASS: Represents an hourglass filter.
    - ELLIPTIC: Represents an elliptic filter.

    Custom, raised-cos, and matched filter types are not available in this release.
```
The suggestions here place **Attributes** in bold to basically treat it a heading. A blank line is needed before the first list item and after the last item. The way you had it formatted, all of the list items would have been printed across in one sentence. (If you don't want to make it a list, you will have to place a blank line between each entry. Please fix the others.